src/components/Auth/Login/Login.react.js

```diff
@@ -104,13 +107,15 @@ class Login extends Component {
 
   handleSubmit = e => {
     e.preventDefault();
-    const { actions, openSnackBar } = this.props;
+    const { actions, openSnackBar, location, history } = this.props;
     const { password, email } = this.state;
 
     if (!email || !password) {
       return;
     }
 
+    console.log(this.props);
```

Comment:
Remove this.

Reply:
@akshatnitd Made the changes you asked for